src/Tribe/Utils/Post_Thumbnail.php

```diff
@@ -286,17 +286,22 @@ public function serialize() {
 		$data            = $this->fetch_data();
 		$data['post_id'] = $this->post_id;
 
-		return serialize( $data );
+		return wp_json_encode( $data );
 	}
 
 	/**
 	 * {@inheritDoc}
 	 */
 	public function unserialize( $serialized ) {
-		$data          = unserialize( $serialized );
+		$data          = json_decode( $serialized, true );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Equals sign not aligned correctly; expected 1 space but found 10 spaces
* :no_entry_sign: TRUE, FALSE and NULL must be uppercase; expected "TRUE" but found "true"

<sub>hash: 89cb0d2414fe3e1da46650c90494df7a</sub>